.github/dependabot.yml

```diff
@@ -1097,8 +1102,3 @@ updates:
     schedule:
       interval: "weekly"
       day: "wednesday"
-  - package-ecosystem: "gomod"
```

Comment:
This change seems unrelated

Reply:
indeed please revert that chunk and I think we'll be ok.

Reply:
actually it's probably not unrelated. We have maxed out our dependabot updates, and this might be removed to make room for this new component (we go by alphabetical order).

Reply:
Closing this thread as answered.